### ISSUE TITLE
Implement .detach()

### ISSUE THREAD

**Issue by ZauberNerd** [garthenweb/jquery-next#24]
http://api.jquery.com/detach/
